lib/Backend/Inline.cpp

```diff
@@ -3007,7 +3062,7 @@ Inline::InlineCallApplyTarget_Shared(IR::Instr *callInstr, bool originalCallTarg
     // instrNext
     IR::Instr* instrNext = callInstr->m_next;
 
-    return InlineFunctionCommon(callInstr, originalCallTargetOpndIsJITOpt, originalCallTargetStackSym, inlineeData, inlinee, instrNext, returnValueOpnd, callInstr, nullptr, recursiveInlineDepth, safeThis, isApplyTarget)->m_prev;
+    return InlineFunctionCommon(callInstr, originalCallTargetOpndIsJITOpt, originalCallTargetStackSym, inlineeData, inlinee, instrNext, returnValueOpnd, callInstr, nullptr, recursiveInlineDepth, safeThis, isApplyTarget, inlineeDefInstr != nullptr)->m_prev;
```

Comment:
>inlineeDefInstr [](start = 220, length = 15)

nit: You might have thought about it as well, but if you pass the `inlineeDefInstr`, you won't have to look for it again in `PrepareInsertionPoint`

Reply:
good call, updated